crates/rooch-rpc-api/src/jsonrpc_types/move_types.rs

```diff
@@ -383,6 +384,78 @@ impl From<H256View> for H256 {
     }
 }
 
+#[serde_as]
+#[derive(Default, Debug, Clone, PartialEq, Eq, Hash, Serialize, Deserialize, JsonSchema)]
+pub struct H64View(
+    #[schemars(with = "Hex")]
+    #[serde_as(as = "Readable<Hex, _>")]
+    [u8; 8],
+);
+
+impl From<H64> for H64View {
+    fn from(value: H64) -> Self {
+        H64View(value.0)
+    }
+}
+
+impl From<H64View> for H64 {
+    fn from(value: H64View) -> Self {
+        H64(value.0)
+    }
+}
+
+#[serde_as]
+#[derive(Default, Debug, Clone, PartialEq, Eq, Hash, Serialize, Deserialize, JsonSchema)]
+pub struct H160View(
```

Comment:
Is the `H160` in JSON need the `0x` prefix?

Reply:
I have changed it to `H176View` since it's 22 bytes adding `0x` prefix.